app/layout.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
```

Comment:
since we're using next, it should be really easy to drop in [geist](https://vercel.com/font#get)